AsyncDisplayKit/ASTableView.mm

```diff
@@ -326,9 +333,14 @@ - (void)beginUpdates
 
 - (void)endUpdates
 {
-  [_dataController endUpdates];
+  [self endUpdatesAnimated:YES completion:nil];
 }
 
+- (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL completed))completion;
+{
+  _endUpdatesAnimationsEnabled = animated;
```

Comment:
In thinking about this now, it seems it's possible for this value to be overwritten by overlapping calls to endUpdates. The best option would be to pass the animated value through the delegates, @appleguy do you think that's the way to go or do you have another suggestion? Turns out the simple approach of using the completion block is not 100% reliable and can lead to animations becoming disabled globally.